infrastructure/src/main/java/org/wso2/testgrid/infrastructure/providers/AWSProvider.java

```diff
@@ -304,6 +305,21 @@ private boolean doRelease(InfrastructureConfig infrastructureConfig, String stac
                         withParameterValue((String) theScriptParameter.getValue());
                 cfCompatibleParameters.add(awsParameter);
             });
+
+            //Set WUM credentials
+            if (expected.getParameterKey().equals("WUMUsername")) {
+                Parameter awsParameter = new Parameter().withParameterKey(expected.getParameterKey()).
+                        withParameterValue(ConfigurationContext.getProperty(ConfigurationContext.
+                                ConfigurationProperties.WUM_USERNAME));
+                cfCompatibleParameters.add(awsParameter);
+            }
+
+            if (expected.getParameterKey().equals("WUMPassword")) {
```

Comment:
Shall we add a constant for the keys?

Reply:
Yeah we can. Did n't add any constants as it is used only once.

Reply:
OK.. I think we'll need to add the WUM credentials to other infrastructure providers as well later..